samples/logs.js

```diff
@@ -88,6 +88,23 @@ async function writeLogEntryAdvanced(logName, options) {
   // [END logging_write_log_entry_advanced]
 }
 
+async function listLogs() {
+  // [START logging_list_logs]
+  // Imports the Google Cloud client library
+  const {Logging} = require('@google-cloud/logging');
```

Comment:
If this got embedded in cloud.google.com, the code inside of the region tag would have a top level await.  That means folks couldn't reasonably copy and paste the sample without making changes.  Instead, can we follow a model that introduces an async wrapper method?  For Example:

```js
async function listLogsMain() {
  // [START logging_list_logs]
  // Imports the Google Cloud client library
  const {Logging} = require('@google-cloud/logging');
  
  // Creates a client
  const logging = new Logging();

  async function listLogs() {
    const [logs] = await logging.getLogs();
    console.log('Logs:');
    logs.forEach(log => {
      console.log(log.name);
    });
  }
  listLogs();
  // [END logging_list_logs]
}
```

It does leave a hanging promise, but at least this way the sample is copy/pastable.